charts/cert-manager-webhook-safedns/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "1.0"
```

Comment:
Does this need to be bumped?

Reply:
Nope - that's just for identifying the version of the underlying app (which isn't changing here)